content/en/apps/tutorials/application-tests.md

```diff
@@ -162,21 +179,29 @@ const result = await harness.fillForm('assessment', ...assessments.cough('3'));
 
 The test files are usually grouped in folders to read and run them easily. One way of grouping them is by creating folders for each of the components that can be tested: forms, contact summary, tasks, and targets.
 
-{{< figure src="tests-dir.png" class="left col-6 col-lg-4">}}
+![alt_text](tests-dir.png "Test directory structure")
 
 ### Testing Forms
-##### What do you test?
-||
-|-|-|
-|Minimum:|Filling a form with the most common options should not result in any error.|
-|Ideal:|All input combinations and constraints are tested with the report fields.|
+<table>
+  <tr>
+    <th colspan='2'> What do you test? </th>
+  </tr>
+  <tr>
+   <td>Minimum:</td>
+   <td>Filling a form with the most common options should not result in any error.</td>
+  </tr>
+  <tr>
+   <td>Ideal:</td>
+   <td>All input combinations and constraints are tested with the report fields.</td>
+  </tr>
+</table>
```

Comment:
Please use markdown tables instead of HTML

```suggestion
| What do you test?  |
|--|--|
|Minimum:|Filling a form with the most common options should not result in any error.|
|Ideal:|All input combinations and constraints are tested with the report fields.|
```